### DISCUSSION TITLE
Welcome to SmallBASIC Discussions!

### DISCUSSION THREAD

**Discussion by chrisws** [smallbasic/SmallBASIC#107] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by informativez** [comment]
Is there a way to read a game controller like a joystick and triggers from an xbox style handheld usb controller in smallbasic?

**Reply by chrisws** [reply]
I haven't tested this yet, but the raylib module should give you access to game controllers.

**Reply by david-hot** [reply]
I just downloaded Smallbasic. I am 71 and used a little basic in 1983. I am hoping to pick it up again.
